etc/c/odbc/sql.d

```diff
@@ -15,7 +15,7 @@ typically not a part of the distribution packages of other compilers
 such as `gdc` and `ldc`). To use ODBC, install it per the vendor- and
 platform-specific instructions and then use the appropriate command-line
 flags (e.g. for dmd, `-L-lodbc` on Posix and `-Lodbc32.lib` on Windows) to link
-with the ODBC library. On Windows, using $(D pragma(lib, "odbc32")) in D
+with the ODBC library. On Windows, using `pragma(lib, "odbc32")` in D
```

Comment:
Only single-word (no spaces) inline code should be in backquotes. This is because many editors reflow comments automatically, and we don't support backquoted code spanning multiple lines.

Reply:
Hmm, smart editors should be able to handle that, but restricted to single-words for now then.

Reply:
You mean, smart editors should be able to handle that once someone writes the code for it? I don't think anything is smart enough to do this correctly right now.